### ISSUE TITLE
[4.0] Cassiopeia landmarks missing

### ISSUE THREAD

**Issue by ceford** [joomla/joomla-cms#35484]
### Steps to reproduce the issue

I have my main menu in position sidebar-left (or sidebar-right). Not using bottom positions but same problem there too.

### Expected result

All seems OK.

### Actual result

Axe accessibility tool tells me "All page content should be contained by landmarks"

**Comment by PhilETaylor** [comment]
I think its also missing a named anchor for `top` that the return to top feature uses... at the moment it returns to top ok with javascript but its missing the actual named anchor somewhere.

**Comment by brianteeman** [comment]
It is technically impossible for the template to apply certain landmarks for a module position as it cannot know what module you will be placing there.

This is why in the advanced module options there is the ability to set the correct tag
![image](https://user-images.githubusercontent.com/1296369/135747055-3ef8749b-d223-48ab-9efa-ab3c9234f588.png)

**Comment by ceford** [comment]
So with a clean install of 4.0.3 I need to set the Main Menu module to nav and the Login module to aside(?). Then Axe just tells me 'Heading levels should increase by 1'. Is all of this documented somewhere? Do we need a tutorial on how to make a site Aria compliant? Should this issue be closed now or left as a reminder to provide or improve documentation?

**Comment by chmst** [comment]
Please let it open as a reminder for documentation.

**Comment by brianteeman** [comment]
Has this been documented. If so it should be closed as there is nothing to do here

**Comment by chmst** [comment]
Documentation is still work in progress. But it is not a Joomla issue, that's correct.
